### PR TITLE
Add binding for onCompleteUpdate, and Init checkUpdateAvailability

### PR DESCRIPTION
**Pull request by ybbond**
## Motivation
- bind already exposed method `completeUpdate` to do functions when update completes.
- add method `checkUpdateStatus` to check whether update available or not
- improve the documentation

## Summary
- [x] bound `completeUpdate` java method as `onCompleteUpdate`
  - [x] update related documentation
- [x] create new java method `checkUpdateStatus`
  - [x] bound the method as react method `checkUpdateAvailability`
  - [x] update related documentation

### CODE REVIEW

File: android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.facebook.react.bridge.WritableNativeMap;
 
+import com.facebook.stetho.inspector.elements.ShadowDocument;
```
Comment (gedeagas) [inline android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java]: This should not be imported.

**Review by gedeagas — CHANGES_REQUESTED** [review]
Minor feedback.

**Comment by gedeagas** [comment]
@ybbond Hi, Thank you for the pr. Would be great if you updated the docs too.

File: android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java (hunk 1 above)
Comment (ybbond) [reply android/src/main/java/com/agastya/androidinappupdates/AndroidInappUpdatesModule.java]: fixed on 73eff8d0916e79c0c3a032d223330f235b104a89

**Comment by ybbond** [comment]
> @ybbond Hi, Thank you for the pr. Would be great if you updated the docs too.

thanks for the input! the docs is will be ready, just need to make sure the native module and bindings work well

File: README.md [hunk 2]
```diff
@@ -51,27 +51,58 @@ try {
 ```
 ## Reference
 ### Methods
-`startUpdateFlow()`
+#### startUpdateFlow(appUpdateType, clientVersionStalenessDays)
```
Comment (gedeagas) [inline README.md]: Hi, @ybbond while it's nice to have the function params on the `h4` i much prefer the complete function with params on the code example itself. 

For reference https://reactnative.dev/docs/appregistry#registercomponent

**Review by gedeagas — CHANGES_REQUESTED** [review]
Minor feedback on the docs

File: README.md (hunk 2 above)
Comment (ybbond) [reply README.md]: I have updated the section title for that method, omitting the params

Comment (gedeagas) [reply README.md]: LGTM

**Comment by gedeagas** [comment]
LGTM.

**Review by gedeagas — APPROVED** [review]

**Pull request merged**